conf/messages

```diff
@@ -974,4 +974,13 @@ returns.list.noTaxDue=You did not owe any tax for this return
 returns.list.leftToPay=Total left to pay
 returns.list.taxOwed=Tax owed for return
 returns.list.sentDate=Sent to HMRC on {0}
-returns.paymentTakes3Days=Payments may take up to 3 working days to be shown on this page
\ No newline at end of file
+returns.paymentTakes3Days=Payments may take up to 3 working days to be shown on this page
+
+disposalDateMixedUseOrIndirect.title=You cannot use this service
+disposalDateMixedUseOrIndirect.p1=You cannot use this service to report Capital Gains on:
+disposalDateMixedUseOrIndirect.p1.reason1=disposals of 'mixed use' UK property
```

Comment:
we use curly quotes for messages instead of straight ones - you can copy and paste the characters from here: https://smartquotesforsmartpeople.com/ or from other quote in that file, though you'd have to look for them a bit in there.